### PR TITLE
Started an IANA Considerations section; expect some edits and additions

### PR DESCRIPTION
**Pull request by paulehoffman**

### CODE REVIEW

File: draft-ietf-dnsop-rfc7958bis.mkd [hunk 1]
```diff
@@ -419,6 +421,16 @@ This document aims to specify carefully the means by which such trust
 anchors are published, with the goal of making it easier for those
 trust anchors to be integrated into user environments.
 
+# IANA Considerations
+
+Each time IANA produces a new trust anchor,
+it SHOULD publish it using the format described in this document.
+
+IANA MAY delay the publication of a new trust anchor for operational reasons,
+such as having sufficient backup of a newly-created key in multiple facilities.
```
Comment (GuillaumeBailey) [inline draft-ietf-dnsop-rfc7958bis.mkd]: What is meant by "sufficient backup"? The phrasing is awkward.

You can also simply strike the example.

File: draft-ietf-dnsop-rfc7958bis.mkd [hunk 2]
```diff
@@ -419,6 +421,16 @@ This document aims to specify carefully the means by which such trust
...
+
+When a trust anchor that was published earlier is no longer valid,
+IANA SHOULD update the trust anchor file to reflect that fact by setting a validUntil date for the trust anchor.
```
Comment (GuillaumeBailey) [inline draft-ietf-dnsop-rfc7958bis.mkd]: I feel like this should be MUST? What is IANA doing if they allow the trust anchor file to reflect an incorrect validUntil date? 

If the notion is that the trust anchor file is not authoritative for validity, then why is validity information in it? 

If the notion is that there may be a delay in reflecting that change or that an old trust anchor file might be maliciously replayed, we should have concrete statements about this being in or out of scope for the protocol and maybe giving guidance about behavior.

**Review by GuillaumeBailey — APPROVED** [review]

File: draft-ietf-dnsop-rfc7958bis.mkd (hunk 1 above)
Comment (paulehoffman) [reply draft-ietf-dnsop-rfc7958bis.mkd]: I think we can remove "sufficient backup of" but leave the example to explain one reason they would delay. Does "...operational reasons, such as having a newly-created key in multiple facilities" work for you?

File: draft-ietf-dnsop-rfc7958bis.mkd (hunk 2 above)
Comment (paulehoffman) [reply draft-ietf-dnsop-rfc7958bis.mkd]: I thought I had a good reason for SHOULD vs. MUST, but now I can't remember it. And I agree that if we have SHOULD we need to say what the less-than-MUST options are. So, let's go with MUST.

File: draft-ietf-dnsop-rfc7958bis.mkd (hunk 1 above)
Comment (ableyjoe) [reply draft-ietf-dnsop-rfc7958bis.mkd]: "IANA MAY delay the publication of a new trust anchor for operational reasons, e.g. to arrange secure transport newly-generated key materials from one facility and to verify an intact chain of custody upon arrival in a second facility."

File: draft-ietf-dnsop-rfc7958bis.mkd [hunk 3]
```diff
@@ -419,6 +421,16 @@ This document aims to specify carefully the means by which such trust
 anchors are published, with the goal of making it easier for those
 trust anchors to be integrated into user environments.
 
+# IANA Considerations
+
+Each time IANA produces a new trust anchor,
+it SHOULD publish it using the format described in this document.
```
Comment (ableyjoe) [inline draft-ietf-dnsop-rfc7958bis.mkd]: "When a new trust anchor produced by IANA is ready for use, IANA MUST publish it using the format described in this document."

File: draft-ietf-dnsop-rfc7958bis.mkd (hunk 2 above)
Comment (ableyjoe) [reply draft-ietf-dnsop-rfc7958bis.mkd]: "When a trust anchor that was previously published is no longer suitable for use, IANA MUST update the trust anchor document accordingly by setting a validUntil date for that trust anchor. The validUntil attribute that is added MAY be a date in the past or in the future, depending on circumstances."

File: draft-ietf-dnsop-rfc7958bis.mkd [hunk 4]
```diff
@@ -419,6 +421,16 @@ This document aims to specify carefully the means by which such trust
 anchors are published, with the goal of making it easier for those
 trust anchors to be integrated into user environments.
 
+# IANA Considerations
+
```
Comment (ableyjoe) [inline draft-ietf-dnsop-rfc7958bis.mkd:440]: I think we need some comment in here that describes how this section and this document relates to the DPS that IANA publishes for management of the root zone KSK. If we need the DPS to be updated to reflect the directions in this section, for example, we should say that.

File: draft-ietf-dnsop-rfc7958bis.mkd (hunk 1 above)
Comment (paulehoffman) [reply draft-ietf-dnsop-rfc7958bis.mkd]: That feels too specific for a "MAY", even though it is what they do today. I'd rather keep it at "having a newly-created key in multiple facilities".

File: draft-ietf-dnsop-rfc7958bis.mkd (hunk 2 above)
Comment (paulehoffman) [reply draft-ietf-dnsop-rfc7958bis.mkd]: Ooooh, good addition. Yes.

File: draft-ietf-dnsop-rfc7958bis.mkd (hunk 3 above)
Comment (paulehoffman) [reply draft-ietf-dnsop-rfc7958bis.mkd]: Agree. A MUST is better than a SHOULD.

File: draft-ietf-dnsop-rfc7958bis.mkd (hunk 4 above)
Comment (paulehoffman) [reply draft-ietf-dnsop-rfc7958bis.mkd:440]: Good call. I'll work something up for the draft, and it will get modified during WG Last Call.

Comment (paulehoffman) [reply draft-ietf-dnsop-rfc7958bis.mkd:440]: Added in 97b1b8b

**Pull request merged**